nagios/config/wavefront-csp-oauth.cfg

```diff
@@ -0,0 +1,45 @@
+# Nagios command definitions for host and service metrics to the Wavefront server using CSP Oauth
```

Comment:
Nagios command definitions for host and service metrics to the Operations for Applications server using server to server OAuth app credentials (id and secret)